### PR TITLE
Refs #104: Handle FK _id field name

### PR DESCRIPTION
**Pull request by hsmett**
Here is a fix to handle foreign keys _id field names.

### CODE REVIEW

**Comment by romgar** [comment]
@hsmett your MR is trying to fix different issues (fkeys, deferred fields and `skip` not useful anymore). Please create individual MRs for each issue.

**Comment by hsmett** [comment]
My appologies :)
Here is a much smaller commit, which only intends to fix the fk_id fields behaviour.

File: tests/test_save_fields.py [hunk 1]
```diff
@@ -67,3 +67,29 @@ def test_save_only_specific_fields_should_let_other_fields_dirty():
...
+def test_handle_foreignkeys_id_field_in_update_fields():
+    tm1 = TestModel.objects.create(boolean=True, characters='dummy')
+    tm2 = TestModel.objects.create(boolean=True, characters='dummy')
+    tmwfk = TestModelWithForeignKey.objects.create(fkey=tm1)
+
+    tmwfk.fkey = tm2
+    assert tmwfk.get_dirty_fields(check_relationship=True) == {'fkey': tm1.pk}
+
+    tmwfk.save(update_fields=['fkey_id'])
+    assert tmwfk.get_dirty_fields() == {}
```
Comment (romgar) [inline tests/test_save_fields.py]: Missing `check_relationship=True` in the assertion.

Comment (hsmett) [inline tests/test_save_fields.py]: updated :)

**Comment by romgar** [comment]
Sounds good to go @hsmett ! Can you please update the `ChangeLog.rst` file in the `master (unreleased)` section?

**Comment by romgar** [comment]
Thanks a lot @hsmett !!

**Pull request merged**